lib/args.js

```diff
@@ -81,28 +82,32 @@ module.exports = function parsedArgs (args) {
   return Object.assign({}, {
     _: parsedArgs._,
     arg: parsedArgs.arg,
-    help: parsedArgs.help,
+    help: toBool(parsedArgs.help),
     path: parsedArgs.path,
     remote: parsedArgs.remote,
     branch: parsedArgs.branch,
     fromCommit: parsedArgs['from-commit'],
     toCommit: parsedArgs['to-commit'],
-    noVerify: parsedArgs['no-verify'],
-    silent: parsedArgs.silent,
-    dryRun: parsedArgs['dry-run'],
+    noVerify: toBool(parsedArgs['no-verify']),
+    silent: toBool(parsedArgs.silent),
+    dryRun: toBool(parsedArgs['dry-run']),
     ghToken,
-    ghReleaseEdit: parsedArgs['gh-release-edit'],
-    ghReleaseDraft: parsedArgs['gh-release-draft'],
-    ghReleasePrerelease: parsedArgs['gh-release-prerelease'],
+    ghReleaseEdit: toBool(parsedArgs['gh-release-edit']),
+    ghReleaseDraft: toBool(parsedArgs['gh-release-draft']),
+    ghReleasePrerelease: toBool(parsedArgs['gh-release-prerelease']),
     ghGroupByLabel: parsedArgs['gh-group-by-label'],
     npmAccess: parsedArgs['npm-access'],
     npmDistTag: parsedArgs['npm-dist-tag'],
     npmOtp: parsedArgs['npm-otp'],
     tag: parsedArgs.tag,
     verbose: parsedArgs.verbose,
     semver: parsedArgs.semver,
-    major: parsedArgs.major
+    major: toBool(parsedArgs.major)
   })
 }
 
+function toBool (value) {
+  return value && Boolean(value)
```

Comment:
How the tests are passing? 🤔 

```
Boolean('false') // true
```

Reply:
Black magic. I wonder if yargs is always string true or false or not. Will have to investigate it furthee

Reply:
Ok, fixed it. the uit test was wrong.